arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -421,3 +479,28 @@ ClusterIndex::coveredFields() const {
       return _fields;
   }
 }
+
+bool ClusterIndex::inProgress() const {
```

Comment:
Is this method called/used?

Also it doesn't look good, because if I will call it for N indexes it will be read Collection indexes N times

Reply:
fixed